src/Npgsql.Json.NET/JsonHandler.cs

```diff
@@ -64,7 +64,7 @@ protected override int ValidateObjectAndGetLength(object value, ref NpgsqlLength
             return base.ValidateAndGetLength(s, ref lengthCache, parameter);
         }
 
-        protected override Task WriteObjectWithLength(object value, NpgsqlWriteBuffer buf, NpgsqlLengthCache? lengthCache, NpgsqlParameter? parameter, bool async)
+        protected override Task WriteObjectWithLength(object? value, NpgsqlWriteBuffer buf, NpgsqlLengthCache? lengthCache, NpgsqlParameter? parameter, bool async)
```

Comment:
Why does this need to be nullable? Aren't nulls checked outside of this method? We should also be able to remove the null check below.

Reply:
As discussed, we'll look at this after this PR.